hardware/card/raid/lsi/h700.pan

```diff
@@ -0,0 +1,9 @@
+structure template hardware/card/raid/lsi/h700;
+
+'bbu' = true;
+'bus' = 'PCI-E';
+'manufacturer' = 'LSI';
```

Comment:
In most of the file, the LSI manufacturer is written lowercase. It is not the case here, it may be worth to use lowercase for consistency.

Reply:
You are right, it should be lowercase.